worker/buildbot_worker/pb.py

```diff
@@ -204,7 +204,7 @@ def tcpConnectionDescr(host, port):
         allowed in DNS, nor any current IP addresses, it can be valid in
         ``/etc/hosts`` entries.
         """
-        return 'tcp:host={}:port={}'.format(
+        return 'tcp:host={0}:port={1}'.format(
```

Comment:
you only need 0/1 in case the arguments are used in a different order.  no need for this change.